website/docs/getting-started.md

```diff
@@ -0,0 +1,153 @@
+:::
+
+## Additional information on Discordeno
+
+Here are some other nice to know things about Discordeno that might interest you. Even if some may be more advanced use-cases.
+
+### Gateway methods
+
+You might want to edit some state like your bot status, for this you will need to use the methods in `bot.gateway`. Those methods will not give you transformed objects then they do give something back and if you want to get transformed objects you will need to call the transformers yourself (the transformers are located on `bot.transformers`).
+
+### OAuth2
+
+Some of the methods in `bot.helpers` will accept a `Bearer` token, or a `Client ID` and `Client Secret`. Those methods are for OAuth2 and will require you to setup the OAuth2 authorization flow with your application. An example of these methods is `bot.helpers.getCurrentUser(bearerToken)`, this is a method that can not be called with a usual Bot token and so will require the OAuth2 token.
+
+Discordeno includes some utilities (such as `createOAuth2Link` to help create a link for the OAuth2 authorization link) and also the methods to do the exchange of the token and refresh of your exiting token.
+
+<!-- TODO: Add a link to the page on how to use OAuth2 in Discordeno. -->
+
+You can read more about OAuth2 in Discord on the [documentation](https://discord.com/developers/docs/topics/oauth2).
+
+### Use individual features from Discordeno separably
+
+While we do provide a `@discordeno/bot` package on npm with all the feature, you might just need a subset of those. For this reason Discordeno is split across multiple packages which are the following:
+
+- `@discordeno/bot`: Groups all other packages and adds transformers and other functionalities on top.
+- `@discordeno/rest`: Will only provide the ratelimit features on the Discord REST Api. This will provide the same object as [`bot.rest`](#raw-rest-methods) when you create a `RestManager`.
+- `@discordeno/gateway`: Will only provide the gateway connection to Discord. This will provide the same object as `bot.gateway` when you create a `GatewayManager`
+- `@discordeno/utils`: Will only provide some utilities that are independent from other Discordeno features.
+- `@discordeno/types`: Will only provide the Discord & Discordeno types used in all other packages.
+
+The use-case for the separate packages may be more advanced compared to the use-cases where it makes more sense to simply use `@discordeno/bot`.
+
+### Raw REST methods
+
+While we recommend using `bot.helpers` you might find yourself in a situation where avoiding the transformers and desired proprieties can be beneficial, for that you can use the `bot.rest` methods, in this object there are the same methods from `bot.helpers` but with a more raw return object. The return objects will simply be the discord raw response but with the keys being camelCase instead of snake_case.
```

Comment:
```suggestion
While we recommend using `bot.helpers`, you might find yourself in a situation where avoiding the transformers and desired properties can be beneficial. For that, you can use the `bot.rest` methods, in this object, there are the same methods from `bot.helpers` but with a more raw return object. The return objects will simply be the raw response from Discord, but with the keys being camelCase instead of snake_case.
```